cedar-policy-core/src/parser/err.rs

```diff
@@ -129,139 +129,132 @@ impl ToASTError {
     }
 }
 
+const POLICY_SCOPE_HELP: &str =
+    "policy scopes must contain a `principal`, `action`, and `resource` element in that order";
+
 /// Details about a particular kind of `ToASTError`.
 #[derive(Debug, Diagnostic, Error, Clone, PartialEq, Eq)]
 #[non_exhaustive]
 pub enum ToASTErrorKind {
```

Comment:
Can you list out which error variants you removed and which you combined? Kind of hard to parse out of the diff.

Reply:
Sorry for being lazy with my original notes 😄 Added the requested info to the PR description.

Reply:
thanks